SQL/New_patches/2018-01-16_map_parameter_type_and_parameter_type_category_for_imaging.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO parameter_type_category_rel (ParameterTypeID, ParameterTypeCategoryID)
+  SELECT ParameterTypeID, ParameterTypeCategoryID
```

Comment:
```suggestion
  SELECT DISTINCT ParameterTypeID, ParameterTypeCategoryID
```

Reply:
@driusan Thank you for the suggestion. Implemented in the next commit. I also renamed the patch with the proper year ;).